test/integration/targets/azure_rm_loadbalancer/tasks/main.yml

```diff
@@ -1,26 +1,31 @@
+- name: Prepare random number
+  set_fact:
+    rpfx: "{{ resource_group | hash('md5') | truncate(7, True, '') }}{{ 1000 | random }}"
+  run_once: yes
+
 - name: create public ip
   azure_rm_publicipaddress:
-    name: ansiblepipstandard
+    name: "pipb{{ rpfx }}"
     sku: Standard
     allocation_method: Static
     resource_group: '{{ resource_group }}'
 
 - name: create public ip
   azure_rm_publicipaddress:
-    name: ansiblepip3
+    name: "pipa{{ rpfx }}"
     resource_group: '{{ resource_group }}'
 
 - name: clear load balancer
   azure_rm_loadbalancer:
     resource_group: '{{ resource_group }}'
-    name: lbtestfromansible
+    name: "lb{{ rpfx }}"
```

Comment:
why not set variables for those names , instead of construct them in each tasks, and remember them in later tasks?